pom.xml

```diff
@@ -72,9 +72,32 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <javac.version>9+181-r4173-1</javac.version>
+    <native-image.version>0.9.11</native-image.version>
+    <graalvm.version>22.0.0.2</graalvm.version>
     <assembly.skipAssembly>true</assembly.skipAssembly>
   </properties>
 
+  <dependencyManagement>
+    <!-- Forcing the versions for Enforcer's dependency convergence rule -->
```

Comment:
I'll think about what to do with these dependency convergence error.

Without these settings, the enforcer rule fails:

```
[INFO] --- maven-enforcer-plugin:3.0.0:enforce (enforce) @ cloud-sql-connector-r2dbc-core ---
Downloading from oss-sonatype-snapshots: https://oss.sonatype.org/content/repositories/snapshots/com/google/cloud/sql/cloud-sql-connector-r2dbc-core/1.5.1-SNAPSHOT/maven-metadata.xml
[WARNING] 
Dependency convergence error for org.graalvm.nativeimage:svm:jar:22.0.0.2:provided paths to dependency are:
+-com.google.cloud.sql:cloud-sql-connector-r2dbc-core:jar:1.5.1-SNAPSHOT
  +-com.google.cloud.sql:jdbc-socket-factory-core:jar:1.5.1-SNAPSHOT:compile
    +-org.graalvm.nativeimage:svm:jar:22.0.0.2:provided
and
+-com.google.cloud.sql:cloud-sql-connector-r2dbc-core:jar:1.5.1-SNAPSHOT
  +-io.netty:netty-handler:jar:4.1.75.Final:compile
    +-io.netty:netty-common:jar:4.1.75.Final:compile
      +-org.graalvm.nativeimage:svm:jar:19.3.6:provided
```